python-sdk/example_dags/example_load_file.py

```diff
@@ -352,7 +357,7 @@
     aql.load_file(
         input_file=File("s3://tmp9/homes_main.csv", conn_id=AWS_CONN_ID),
         output_table=Table(
-            conn_id=MSSQL_CONN_ID,
+            conn_id=MYSQL_CONN_ID,
```

Comment:
Why did we replace the `MSSQL_CONN_ID`(MS SQL)  by a MySQL one?

Reply:
@vatsrahul1001 the names are very similar, but MySQL != MSSQL.

MySQL is an open-source relational database, while MSSQL Server is a Microsoft-developed db.

Reply:
@tatiana i have updated MSSQL